frontend/src/metabase-lib/expressions/format.js

```diff
@@ -111,9 +111,7 @@ function formatFunction([fn, ...args], options) {
   }
   const formattedName = getExpressionName(fn);
   const formattedArgs = args.map(arg => format(arg, options));
-  return args.length === 0
-    ? formattedName
-    : `${formattedName}(${formattedArgs.join(", ")})`;
+  return `${formattedName}(${formattedArgs.join(", ")})`;
```

Comment:
Any particular reason for this change?

Reply:
Answered https://github.com/metabase/metabase/pull/26548#discussion_r1028324891